src/services/color/color.service.ts

```diff
@@ -91,7 +83,12 @@ export class ColorService {
     }
 
     getColor(color: ColorIdentifier): ThemeColor {
-        return this._colors[color];
+        return this._colors[color.toLowerCase()];
+    }
+
+    setTheme(theme: string[]) {
+        this._theme.push.apply(this._theme, theme);
```

Comment:
I think you can just assign this.

Reply:
The reason I did it this way was so that the default colors would still be included when using another theme, for example in preexisting applications which switch to the Micro Focus theme we don't want to break uses of old colors throughout the application

Reply:
We don't want the old theme colors to appear after we have installed the Quantum stylesheet and set up the new color set though. Then we would have HPE green appearing wherever 'primary' was specified.

How about extending the color set data to be a map of [color name] -> [class name]? Then you can add in all of the UX Aspects color names and specify their rebranded color, while still allowing the new color names to be used in new code.

````
  "primary": "brand-blue",
  "chart1": "emerald",
````

Or let me know if you have any other ideas.

Reply:
Yes that way sounds good, my thinking was that the variables file would be updated and mapped to the Micro Focus colors therefore primary would be the equivalent of brand blue, in the documentation we would encourage the use of the Micro Focus naming convention with the variables file there as a fallback. I think your approach seem cleaner however.